ResearchKit/Common/CEVRKTheme.h

```diff
@@ -0,0 +1,24 @@
+//
+//  CEVRKTheme.h
+//  ResearchKit
+//
+//  Created by Eric Schramm on 7/10/19.
+//  Copyright © 2019 researchkit.org. All rights reserved.
+//
+
+@import UIKit;
+
+static NSString * _Nonnull const kThemeAllOfUs = @"AllOfUs";
```

Comment:
I think the more idiomatic way to declare this would be
`ORK_EXTERN NSString *const CEVRKThemeAllOfUs ORK_AVAILABLE_DECL;`  
and in the .m file assign the value using  
`NSString *const CEVRKThemeAllOfUs = @"AllOfUs";`

Though would it make the overall implementation easier to declare this as an enum, with two values (Default and AllOfUs)? RKStudio-Participant or CEVResearchKit could add a Swift extension to init the enum using the string value coming from the server.

Reply:
Agreed - Given the idea/push to move logic inside the object, it feels like enum is the way to go.